lib/lndclient/LndClient.ts

```diff
@@ -103,6 +103,22 @@ class LndClient extends BaseClient {
     });
   }
 
+  private unaryCallNative = <T, U>(methodName: string, params: T): Promise<U> => {
```

Comment:
This is almost the same code as `unaryCall` - basically unaryCall can just call this method and return as toObject if we want to keep that functionality. Not urgent or needed for this PR, but just noting it.